pysmt/cmd/install.py

```diff
@@ -547,46 +153,54 @@ def print_welcome():
 def main():
     options = parse_options()
```

Comment:
If no arguments are given, I think we should print the help message. Currently, the script exits without say anything.